iothub_client/src/iothubtransport_amqp_connection.c

```diff
@@ -146,17 +146,17 @@ static void on_connection_state_changed(void* context, CONNECTION_STATE new_conn
 
 static void on_cbs_open_complete(void* context, CBS_OPEN_COMPLETE_RESULT open_complete_result)
 {
-    (void)context;
     (void)open_complete_result;
     if (open_complete_result != CBS_OPEN_OK)
     {
         LogError("CBS open failed");
+		update_state((AMQP_CONNECTION_INSTANCE*)context, AMQP_CONNECTION_STATE_ERROR);
     }
 }
 
 static void on_cbs_error(void* context)
 {
-    (void)context;
+	(void)context;
```

Comment:
please revert this change, it's not needed.

Reply:
Same here, no change and using tab.